simGL/simGL.py

```diff
@@ -3,6 +3,11 @@
 from itertools import combinations
 from scipy.stats import binom
 
+def e2q(e):
+    return -10*np.log(e)
+
+def q2e(q):
+    return np.exp(-q/10)
```

Comment:
Are these for phred-scale base quality scores? If so, I think they should be log base 10.